components/rdv/AddRdvModal.tsx

```diff
@@ -39,7 +39,7 @@ const AddRdvModal = ({
   }, [fetchJeunes, jeuneInitial])
 
   const creneauIsValid = () =>
-    creneau !== '' && creneau.match(/[0-9][0-9]:[0-9][0-9]/gm)
+    creneau !== '' && creneau.match(/^[0-9][0-9]:[0-9][0-9]$/gm)
```

Comment:
```suggestion
    creneau !== '' && creneau.match(/^[0-2][0-9]:[0-5][0-9]$/gm)
```
Pour éviter les horaires type 19:90

Reply:
Si on est des fifou on peut même faire `/^([0-1][0-9]|2[0-3])$/`